content/integrations/sso/sso-jumpcloud.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "JumpCloud SSO"
+#date: 2018-12-11
+draft: false
+tags: ["#sso", "#integrations", "#authentication", "#jumpcloud"]
+author: Lawrence Lane
+weight: 2
+---
+
+{{% alert success %}}
+Single Sign-On (SSO) is a **paid-only feature**. Wish to upgrade? Contact your [Sales Representative](mailto:sales@metricly.com).
+{{% /alert %}}
+
+## 1. Generate a Certificate in Metricly
+
+1. Navigate to **Account Profile** > **SSO**.
+2. Select **Generate**.
+![generate](/images/_index/generate.png)
+3. Copy the generated certificate and use it to create a `.cert` file.
+4. Select **Continue**.
+
+Keep this tab open and open a new tab; you must now login to JumpCloud and upload the certificate.
+
+## 2. Create New Application in JumpCloud
+1. Log into **JumpCloud** as an administrator.
+{{% notice tip %}}
+ Login page _must_ say **JumpCloud Administrator Login** and _not_  **JumpCloud User Login**.
+{{% /notice %}}
+2. Navigate to **Applications** > **+** (New Application). A slide-out panel appears.
+![jc-new-application](/images/sso-jumpcloud/jc-new-application.png)
+3. Search for `SAML` and select **CustomSAML_App** > **Configure**.
+4. Input the following values:
+  - **DISPLAY LABEL**: `Your SSO Label` (whatever string you want)
+  - **IDP ENTITY ID**: `support.netuitive.com`
```

Comment:
The value for this property is arbitrary but choosing it for them would be okay in my opinion. It should read "jumpcloud-metricly" instead of "support.netuitive.com".